src/testdir/test_channel.vim

```diff
@@ -1775,7 +1775,7 @@ endfunc
 
 func Test_job_start_in_timer()
   CheckFeature timers
-  CheckFeature reltimefloat
+  CheckFunction reltimefloat
```

Comment:
Is there a way to detect such mistake automatically?

I just tried adding this in `CheckFeature()` in check.vim:
```
   call assert_match('[-+]' .. a:name .. '\>',
              \      execute('version'),
              \      a:name .. ' is not a feature')
```
It does detect that `reltimefloat` is not a feature
(misused in `Test_job_start_in_timer()`). But unfortunately 
it gives spurious errors saying that "clipboard_working is not
a feature" in some tests.